react-frontend/client/services/APIServices.js

```diff
@@ -0,0 +1,25 @@
+import fetch from 'fetch'
+
+const getFetchRequest = (url) => {
+    return fetch(url)
+    .then(function(response) {
+        return response.json();
+    });
+}
+
+const postFetchWithOptions = (url, options = {}, data = {}) => {
+    return fetch(url, {
+        method: 'POST',
+        ...options,
+        body: JSON.stringify(data),
+    })
+    .then(response => {
+        response.json()
```

Comment:
return `response.json()`;

Reply:
APIServices is a single place for making requests to the node server so if we want to change how we talk to the backend we only have one place to do so.